Makefile.toml

```
@@ -126,7 +126,7 @@ workspace = false
 install_script = ["(cd ./api_tests; yarn install;)"]
 script = [
 '''
-(cd api_tests; yarn run prettier --write '**/*.js')
+(cd api_tests; yarn run prettier --write '**/*.[tj]s')
```

I think we can go for duplication here for the benefit of clarity :D

```suggestion
(cd api_tests; yarn run prettier --write '**/*.{ts,js}')
```